packages/babel-plugin-transform-block-scoping/src/index.ts

```diff
@@ -133,7 +133,10 @@ export default declare((api, opts: Options) => {
               transformBlockScopedVariable(headPath, state, tdzEnabled);
             }
 
-            varPath.get("declarations.0.init").unwrapFunctionEnvironment();
+            const declarations = varPath.get("declarations");
+            declarations[declarations.length - 1]
+              .get("init")
+              .unwrapFunctionEnvironment();
```

Comment:
Nit, but we can simplify this back to what it was if we push the new variables to the end of `varPath` instead of unshifting them.

Reply:
Do you think it would look better if we put the variable first?

Reply:
It would look similar, but it's slightly simpler code on our end :)

Reply:
Sounds very reasonable!